src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -437,6 +458,10 @@ export default {
 					singleConversation: false,
 				})
 				this.isFetchingConversations = false
+				this.handleUnreadMention()
+				if (!this.$route.params.token) {
+					this.scrollBottomUnread()
```

Comment:
SHould not do this

Reply:
Currently, when you visit page, it will scroll down to unread mentions.
If we remove those code, it will not auto scroll anymore.
So which is the right behavior ?

Reply:
Yeah no scrolling is expected. We just show the button in that case.